spyder/app/tests/test_mainwindow.py

```diff
@@ -1784,5 +1792,33 @@ def test_render_issue():
     assert test_traceback in test_issue_2
 
 
+@flaky(max_runs=3)
+def test_custom_layouts(main_window, qtbot):
```

Comment:
Please mark this test as slow with the following decorator:

`@pytest.mark.slow`

Reply:
That will make this test to only run in our CIs and not locally. That's better because most main window tests are quite consuming.

Reply:
Yes, I forgot to re-add it. Thanks